Automattic/jetpack
# Components: Fix IconTooltip Popover

## Proposed changes:
Seems like there was some changes in the Popover component and it started using portals. Which meant that it would embed it in a different position in the page breaking some of the styles we apply. This portals can be avoided by using `inline: true` argument in the popover. This PR does just that.

### Other information:


## Jetpack product discussion
pc9hqz-2ie-p2#comment-1829

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Go to the boost in WP admin
* The pricing table should be visible
* Click on one of those info tooltip icons.
* The popovers should be properly styled.

<img width="936" alt="image" src="https://github.com/Automattic/jetpack/assets/3737780/daed018f-aeb2-4d07-a0f2-b01b3c4c57d0">
